DQM/PhysicsHWW/src/HypDilepMaker.cc

```diff
@@ -122,35 +116,35 @@ void HypDilepMaker::SetVars(HWW& hww, const edm::Event& iEvent, const edm::Event
       vector<LorentzVector>  temp_jets_p4;      
 
 	
-      for(unsigned int i = 0; i<jets_p4->size(); i++) {
+      for(unsigned int i = 0; i<jets_p4.size(); i++) {
 	
         // we don't want jets that overlap with electrons
         bool overlapsWithLepton = false;
-        if(!testJetForLeptons(jets_p4->at(i), mus_p4->at(loose_index))) 
```

Comment:
given the index of the loop is hyper safe to use [i] instead of .at(i)
``std::vector<T>::at(..)'' should be banned.
I suggest to trap its use in some code analyzer